README.md

```diff
@@ -1,2 +1,54 @@
-# ASB_Tutorial
-Repository for 2020 American Society of Biomechanics Tutorial
+# ASB 2020 GitHub tutorial setup
+#### [Ryan Alcantara](https://twitter.com/Ryan_Alcantara_) & [Gary Bruening](https://twitter.com/garebearbru) & [Ross Wilkinson]
+
+This will be the repository that we will use for the Git Tutorial for [ASB 2020](https://asb2020.gatech.edu/)!
+
+We will keep updating this as more things get added.
+
+# Pre-Meeting Setup
+This section is intended to take you through the steps required before the meeting. 
+
+The first program everyone will need is some version of Git. Git is the version control software that allows us to see, track, and edit changes to code bases. 
```

Comment:
Keep the language directed towards the user for consistency. Instead of "Git is the version control software that allows *us* to see...", try "Git is the version control software that allows *you* to see...". This goes for the rest of the readme.